### DISCUSSION TITLE
Support Jinja >= 3.0.0

### DISCUSSION THREAD

**Discussion by msabramo** [datapane/datapane#126] [Ideas]
I'm having trouble getting `datapane` to co-exist in a virtualenv with other packages such as `dtale`.

This is seemingly because datapane requires `jinja2 < 3.0.0`:

```$ pipenv graph
...
datapane==0.11.8
...
  - jinja2 [required: >=2.10.0,<3.0.0, installed: 3.0.1]
```

This limits the flexibility of using datapane in projects. For example, I am using `dtale` and dtale requires `Flask`, which requires `Jinja2 >= 3.0`, so I may end up having to remove `datapane` because of this conflict.

It looks like datapane makes pretty minimal use of jinja2, so I wonder if it is a trivial change to test datapane with a newer version of jinja2 and then update [`pyproject.toml`](https://github.com/datapane/datapane/blob/master/pyproject.toml#L67), which has:

```toml
jinja2 = "^2.10.0"
```

**Comment by mands** [comment]
Hi there,

Thanks for raising this - we should be able to move to jinja 3.x easily internally, however we wouldn't want to cause the opposite issue. i.e. blocking users who have other software that depends on jinja 2.x. As such we need to figure out a way to support both v2 and v3 simultaneously for now.

Hopefully will have a fix for that out soon!
